tests/foreman/api/test_user.py

```diff
@@ -445,8 +485,20 @@ def test_negative_create_invalid_length_ssh_key(self):
 
         :CaseImportance: Critical
         """
+        invalid_length_key = 'ssh-rsa AAAAB3NzaC1yc2EAAAABIwAAAQEA5mX2JV6T6CVCRmTlC9vrpgEQp259XQYBKIkhWeHtJXGe8bV6RWxR07deny/coS3OZ0TcM8nXb3J3mykaF1xAK+9nXljcIJy76Us9bjXS25UVXQ18DMOQ0mKR3u29jYjw0TCGBUZxjpOTiUudCoAI7i3XHanQz0vyQaKOhDa1fDb3Qc4hKujJYf76J/XPYpb5DHYcWxELgCFo5MWRnYex2L92gm1xcx1CAPVN5JbpIxhkHuXIJx0JBhJAHUvYopByK0ZNFqMkgylVwvuY2gmo085cO6I5GL23NnaKjLKJo7LLPQpJZ1BscMIyrwQTm2CAvJMok6vh88WsB/TI7XXXXX== ares@bart' # noqa
```

Comment:
Please create separate files with keys in `tests/foreman/data/` folder and read them in tests, hardcoding such strings is a bad style (and especially 1-linear strings, in worst case you could at least use [multi-line strings](https://stackoverflow.com/a/10660443) )